src/cryptoadvance/specter/server_endpoints/wallets/wallets_api.py

```diff
@@ -527,8 +527,9 @@ def addressinfo(wallet_alias):
             match = re.search(derivation_path_pattern, descriptor)
             if not match:
                 logger.debug(
-                    f"Derivation path of this descriptor {descriptor} could not be parsed. Sth. wrong with the regex pattern which was {derivation_path_pattern}?"
+                    f"Derivation path of this descriptor {descriptor} for address {address} could not be parsed. Sth. wrong with the regex pattern which was {derivation_path_pattern}?"
                 )
+                return jsonify(success=False)
             logger.debug(f"This is the derivation path match: {match.group()}")
```

Comment:
I would check whether match is not None here:

```
if match:
    logger.debug(f"This is the derivation path match: {match.group()}")
    derivation_path = "m" + match.group()

```

Reply:
I think both is fine here, since a bool(Match object) == True.
```
def search(pattern, string, flags=0):
    """Scan through string looking for a match to the pattern, returning
    a Match object, or None if no match was found."""
    return _compile(pattern, flags).search(string)
```